install/install.py

```diff
@@ -18,13 +18,19 @@ class Installer(object):
     def __init__(self, RootDir):
         self.RootDir = RootDir
         self.pid = os.getpid()
-        self.scripts_path = os.path.join(RootDir, "scripts") # custom scripts
-        self.owtf_pip = os.path.join(RootDir, "install", "owtf.pip") # OWTF python libraries
-        self.restricted_cfg = os.path.join(RootDir, "install", "distro-independent.cfg") # Restricted tools and dictionaries which are distro independent
-        self.distros_cfg = os.path.join(RootDir, "install", "linux-distributions.cfg") # Various distros and install scripts
+        self.scripts_path = os.path.join(RootDir, "scripts")  # custom scripts
+        self.owtf_pip = os.path.join(RootDir, "install", "owtf.pip")  # OWTF python libraries
+        # Restricted tools and dictionaries which are distro independent
+        self.restricted_cfg = os.path.join(RootDir, "install", "distro-independent.cfg")
+        # Various distros and install scripts
+        self.distros_cfg = os.path.join(RootDir, "install", "linux-distributions.cfg")
         self.parser = argparse.ArgumentParser()
-        self.parser.add_argument('--no-user-input', help='run script with default options for user input', action="store_true")
-        self.parser.add_argument('--core-only', help='install only owtf dependencies, skip optional tools', action="store_true")
+        self.parser.add_argument('--no-user-input',
```

Comment:
Please don't use that line-breaking style. Something like this would be better:
```python
foo(
    bar1,
    bar2)
```
If it can't fit under 120 characters.